composer.json

```diff
@@ -18,16 +18,13 @@
 		"php": ">=5.4",
 		"sirbrillig/phpcs-variable-analysis": "^2.8.3",
 		"squizlabs/php_codesniffer": "^3.5.5",
-		"wp-coding-standards/wpcs": "^2.3"
+		"wp-coding-standards/wpcs": "^2.3",
+		"dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7"
```

Comment:
Let's [normalize](https://github.com/ergebnis/composer-normalize) this into alphabetical order please.

Reply:
In that case, would it make sense to add that package to `require-dev` and add the `--dry-run` check to the Travis `lint` stage (with Composer script to ensure the same settings are used locally and in CI) ?

Reply:
Ok, just checked - adding it to `require-dev` is not an option as the PHP requirements would conflict. Still, we could just install it temporarily for the CI dry-run only.

Reply:
Either way, I've updated the commit to "normalize" the package order for now.

Reply:
I'm not fussed about adding it to the dev dependencies, or having Travis fail a build because of it - it's rare to add new dependencies, so if we happen to catch it manually, great, but if not, not a biggie.